charts/consul/values.yaml

```diff
@@ -263,6 +263,46 @@ global:
         additionalConfig: |
           {}
 
+      controller:
+        # The directory that Kubernetes will use on Kubernetes CRD creation, 
+        # deletion, and update, to get TLS certificates used issued from vault
+        # to send webhooks to the controller.
+        tlsCert:
+          # @type: string
+          secretName: null
+          
+        # The directory that Kubernetes will use on Kubernetes CRD creation, 
+        # deletion, and update, to get CA certificates used issued from vault
+        # to send webhooks to the controller.
+        caCert:
+          # @type: string
+          secretName: null
+      
+      # The Vault role for consul controller to read the Consul's controllers's CA Certificate (unauthenticated).
```

Comment:
this role is needed for reading CA as well as webhook cert/private key, correct?

Reply:
yes.  great call.